src/pages/settings/Security/CloseAccountPage.tsx

```diff
@@ -59,21 +55,21 @@ function CloseAccountPage(props) {
         hideConfirmModal();
     };
 
-    const showConfirmModal = (values) => {
+    const showConfirmModal = (values: OnyxFormValuesFields<typeof ONYXKEYS.FORMS.CLOSE_ACCOUNT_FORM>) => {
         setConfirmModalVisibility(true);
         setReasonForLeaving(values.reasonForLeaving);
     };
 
     /**
      * Removes spaces and transform the input string to lowercase.
-     * @param {String} phoneOrEmail - The input string to be sanitized.
-     * @returns {String} The sanitized string
+     * @param phoneOrEmail - The input string to be sanitized.
+     * @returns The sanitized string
      */
-    const sanitizePhoneOrEmail = (phoneOrEmail) => phoneOrEmail.replace(/\s+/g, '').toLowerCase();
+    const sanitizePhoneOrEmail = (phoneOrEmail: string): string => phoneOrEmail.replace(/\s+/g, '').toLowerCase();
 
-    const validate = (values) => {
+    const validate = (values: OnyxFormValuesFields<typeof ONYXKEYS.FORMS.CLOSE_ACCOUNT_FORM>): Errors => {
         const requiredFields = ['phoneOrEmail'];
-        const userEmailOrPhone = props.formatPhoneNumber(props.session.email);
+        const userEmailOrPhone = formatPhoneNumber(session?.email ?? '');
```

Comment:
Why wouldn't we have an email here?

Reply:
Since `session` goes from Onyx it has `null` in the typing. Email is also option field in the session, so to prevent TS error we need to add default value, but I can try to update `formatPhoneNumber` typing instead, let me try

Reply:
@flodnv Done, what do you think?

Reply:
As a noob, I want to try to understand more about this. I wanted to spin up a discussion in #expensify-open-source  in Slack and tag you but I failed to find you on Slack, what is your handle?

Reply:
@flodnv You can find me as `Viktoryia Kliushun`, I've also pinged you in Slack so you should see me there
@fabioh8010 It would be great to here your thoughts on this one since we have doubts

Reply:
@flodnv In this component `session` is a Onyx prop supplied by `withOnyx` and therefore can be `null` as any other Onyx value, so we can either default the value to `''` or change `formatPhoneNumber` to accept and handle undefined values too. Looks like @VickyStash already changed `formatPhoneNumber` to support it and I think it's the best approach, wdyt?